tests/ops/util/test_consent_util.py

```diff
@@ -1196,7 +1196,6 @@ def custom_data_use(self, db):
                 name="New data use",
                 description="A test data use",
                 parent_key=None,
-                is_default=True,
```

Comment:
noting here for posterity: i removed this because it didn't seem intended and was causing problems 👍